packages/utils/src/v2/TableId.ts

```diff
@@ -0,0 +1,40 @@
+import { arrayToHex } from "./arrayToHex";
+import { bytesToString } from "./bytesToString";
+import { stringToBytes16 } from "./stringToBytes16";
+
+export class TableId {
+  namespace: string;
+  name: string;
+
+  constructor(namespace: string, name: string) {
+    this.namespace = namespace;
+    this.name = name;
+  }
+
+  toString() {
+    return `TableId<${this.namespace || "[empty]"}:${this.name || "[empty]"}>`;
+  }
+
+  toHexString() {
+    return TableId.toHexString(this.namespace, this.name);
+  }
+
+  static toHexString(namespace: string, name: string) {
+    const tableId = new Uint8Array(32);
+    tableId.set(stringToBytes16(namespace), 0);
+    tableId.set(stringToBytes16(name), 16);
+    return arrayToHex(tableId);
+  }
+
+  static fromBytes32(tableId: Uint8Array) {
+    // assumes tableId is a 32-byte hex string, otherwise it left-pads with zeros (for numbers)
+    // this is scary, since zero padding is different depending on the type (bytes types vs number types)
+    // TODO: fix this after we move tableIds to bytes32 instead of uint256
```

Comment:
Should we add this TODO to #444?

Reply:
added https://github.com/latticexyz/mud/issues/484